vmm/Cargo.toml

```diff
@@ -51,6 +51,7 @@ vm-memory = { version = "0.5.0", features = ["backend-mmap", "backend-atomic", "
 vm-migration = { path = "../vm-migration" }
 vm-virtio = { path = "../vm-virtio" }
 vmm-sys-util = { version = ">=0.5.0", features = ["with-serde"] }
+uuid = "0.8"
```

Comment:
nit: We try and keep dependences in this file in alphabetical order.

Reply:
Done.